script-docs/hub/guide/access-rights.rst

```diff
@@ -0,0 +1,27 @@
+==================
+Set access rights
+==================
+
+This section provides guidance on managing users in the Hub.
+
+The Hub allows you to set access rights at two levels: global and scoped. To begin, click the "Account" icon in the upper right corner of the screen, then select "Settings." From the left panel, choose "User Management."
+
+.. image:: /_static/images/hub/access-settings.png
+   :align: center
+   :alt: "Access rights"
+
+Global permissions apply access rights across all projects. You can configure Create, Read, Edit, and Delete permissions for each page or entity. Additionally, for features like the Playground, API Key Authentication, and Permission, you can enable or disable the users’ right to use it.
+
+.. image:: /_static/images/hub/access-permissions.png
+   :align: center
+   :alt: "Set permissions"
+
+Scoped permissions allow for more granular control. For each project, you can specify which pages or entities users are allowed to access.
```

Comment:
Missing a concrete example of someone having read on everything in a project, and edit on a specific dataset